modules/cluster/main.tf

```diff
@@ -8,6 +8,7 @@ resource "null_resource" "create_cluster" {
 
   triggers = {
     cluster_name   = var.cluster_name
+    host_zones     = join(";", var.availability_zones)
```

Comment:
This was getting passed into the module joined. Should I pass a list in instead?

Reply:
In the our modules I converted to passing around an array instead so when I check in those changes it will already be in the correct type.